### PR TITLE
Doc on how to write a Coalton test suite

### PR DESCRIPTION
**Pull request by Izaakwltn**
I wrote a simple guide to writing a test suite for Coalton.

### CODE REVIEW

File: docs/implementing-tests.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    (+ x 20))
+    
+  (define (baz-it string)
+    (coalton-library/string:concat "baz-" string)))
+```
+## Defining the test package
+
+When defining a test package for a Coalton project, you'll actually make two separate packages, one for native Coalton tests, and another that will alias your Coalton tests into a Fiasco test package (in addition to any Common Lisp specific tests you want to implement).
+
+### First you will define the Fiasco test package for your project:
```
Comment (stylewarning) [inline docs/implementing-tests.md:26]: This is a little bikesheddy but could you not put verbiage like this in section headings (here and elsewhere), and instead make it paragraph content? Maybe something like:

```
### Step 1: Define a package

First you will define ...
```

**Review by stylewarning — CHANGES_REQUESTED** [review]

File: docs/implementing-tests.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Defining a Coalton test-package requires `#:coalton-testing`, a package which facilitates Coalton-Fiasco compatibility.
+
+In order to use `#:coalton-testing`, either run `(ql:quickload :coalton/testing)` or add `#:coalton/testing` to your project's .asd file.
+```
+(defpackage #:foobar-native-tests
+  (:use #:coalton-testing #:foobar))
+```
+### Step 3: Initialize Fiasco
+
+Within the Coalton test-package, initialize Fiasco compatibiltiy by calling `#'coalton-fiasco-init` with your Fiasco test-package.
```
Comment (macrologist) [inline docs/implementing-tests.md:43]: The hypen isn't needed

Comment (Izaakwltn) [reply docs/implementing-tests.md]: Sounds good, changed throughout

**Comment by eliaslfox** [comment]
Closing this in favor of #874 which is more comprehensive.

**Pull request closed without merging**